packages/config/src/api/client.js

```diff
@@ -11,7 +11,7 @@ const getApiClient = function ({ token, offline, testOpts = {}, host, scheme, pa
   }
 
   // TODO: find less intrusive way to mock HTTP requests
-  const parameters = removeFalsy({ scheme: testOpts.scheme || scheme, host: testOpts.host || host, pathPrefix })
+  const parameters = removeUndefined({ scheme: testOpts.scheme || scheme, host: testOpts.host || host, pathPrefix })
```

Comment:
The difference: empty strings are not removed anymore, only `undefined` and `null`.
This is only used for testing purpose and never to an empty string, so no problems there.